lib/libzpool/uzfs_mgmt.c

```diff
@@ -462,6 +463,16 @@ uzfs_create_dataset(spa_t *spa, char *ds_name, uint64_t vol_size,
 	return (error);
 }
 
+int
+is_rebuild_cloned_volume(const char *ds_name)
```

Comment:
is_internally_created_clone_volume

Reply:
this fn seems to return 0 if given name is for rebuild cloned volume. We may need to change the return value, and description would be better.